examples/ex1.cpp

```diff
@@ -26,16 +26,17 @@
 //               ex1 -m ../data/mobius-strip.mesh -o -1 -sc
 //
 // Device sample runs:
-//               ex1 -pa -d cuda
-//               ex1 -pa -d raja-cuda
-//               ex1 -pa -d occa-cuda
-//               ex1 -pa -d raja-omp
-//               ex1 -pa -d occa-omp
-//               ex1 -pa -d ceed-cpu
-//               ex1 -pa -d ceed-cuda
-//               ex1 -m ../data/beam-hex.mesh -pa -d cuda
-//               ex1 -m ../data/beam-tet.mesh -pa -d ceed-cpu
-//               ex1 -m ../data/beam-tet.mesh -pa -d ceed-cuda:/gpu/cuda/ref
+//               ex1 -al pa -d cuda
```

Comment:
If we agree on this command line option, we should propagate it to all other examples using `-pa`.

Reply:
Sorry, this has been discussed with @tzanio I will revert and remove `ElementAssembly` from `ex1` and `ex1p`. I will also revert `ex9` and `ex9p` to use two booleans. I was considering keeping the string function, because I find it potentially useful, any opinion on that?